src/Models/Diffusion/charmed.m

```diff
@@ -205,7 +205,11 @@
             
             %% FITTING (with rician assumption)
             if ~moxunit_util_platform_is_octave
-                [xopt, residue] = fmincon(@(x) double(-2*sum(scd_model_likelihood_rician(datadif,max(eps,S0.*equation(obj, addfixparameters(obj.st,x,fixedparam))), SigmaNoise))), double(obj.st(~fixedparam)), [], [], [],[],double(obj.lb(~fixedparam)),double(obj.ub(~fixedparam)),[],optimoptions('fmincon','MaxIter',20,'display','off','DiffMinChange',0.03));
+                h=optimset('Algorithm', 'active-set', 'Display', 'iter', 'MaxIter',100,...
```

Comment:
@tanguyduval reviewing this now. Out of curiosity, what's the expected behaviour of the fitting if running on Octave? You have an `if (is not Octave)` block here, but no else, so this the behaviour for Octave isn't clear to me. I know this is unrelated to your changes in this PR, I'm just curious.

Reply:
Actually, @agahkarakuzu has already started to review this branch and is further along than I am, so I will assign him instead.

Reply:
@agahkarakuzu any update?